Dockerfile

```diff
@@ -0,0 +1,17 @@
+#
+# Build stage
+#
+FROM maven:3.9.5-eclipse-temurin-17-focal AS build
+COPY src /home/app/src
+COPY pom.xml /home/app
+RUN mvn -f /home/app/pom.xml clean package
+RUN ls /home/app/target/
```

Comment:
unnecessary line that can be removed :)

Reply:
I'm confused. You marked 4 lines and wrote "Unnecessary line". 
Should I remove one line or the marked lines?
Sorry for this stupid question

Reply:
Oh. Sorry, that's by mistake.

I only meant `RUN ls /home/app/target/`. The other ones _are_ necessary. :D